README.md

```diff
@@ -3,9 +3,6 @@
 
 This is the Go SDK for Oracle Cloud Infrastructure. This project is open source and maintained by Oracle Corp. 
 The home page for the project is [here](https://godoc.org/github.com/oracle/oci-go-sdk/).
->***WARNING:***: To avoid automatically consuming breaking changes if we have to rev the major version of the Go SDK, 
```

Comment:
The warning here is needed, which is used for reminding user to be aware of consuming breaking changes. We can remove go dep, but vendor the SDK is still an option.

Reply:
But that's just it, they _won't_ get breaking changes if they use go modules. I think that's true of vendoring as well--if they vendor a particular major version, they aren't getting breaking changes (unless you push a breaking change w/o a major version bump). At least, I think that's how vendoring works, I haven't personally played with it myself much.

Reply:
If users use go path(go get -u....), they would consume breaking changes automatically, we prefer to keep the warning, maybe we can move this under go path installation

Reply:
No, they wouldn't, because `-u` would only pull down the latest within that particular major version.

Here for example I installed the module, then I did a `-u` and it didn't update to the `v40` code, then I downloaded the `v40` code and go basically treats it as a completely separate module.
```
~ ❯❯❯ go get -d github.com/oracle/oci-go-sdk@v25.0.0
go get: github.com/oracle/oci-go-sdk@v25.0.0: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v25
~ ❯❯❯ go get -d github.com/oracle/oci-go-sdk                               
~ ❯❯❯ go get -v -d github.com/oracle/oci-go-sdk
~ ❯❯❯ go get -v -u github.com/oracle/oci-go-sdk
github.com/oracle/oci-go-sdk
~ ❯❯❯ go get -v github.com/oracle/oci-go-sdk/v40@v40.0.0
go: downloading github.com/oracle/oci-go-sdk/v40 v40.0.0
github.com/oracle/oci-go-sdk/v40
~ ❯❯❯ go get -v -u github.com/oracle/oci-go-sdk/v40
github.com/oracle/oci-go-sdk/v40
```

Reply:
I should add that I may be ignorant of some `go` cmd tooling, so if I'm wrong, please educate me. This is just what I understand from reading the docs + playing with it locally.